js/dogecoin-i18n.js

```diff
@@ -18,10 +18,10 @@ function getUrlParameter(sParam)
 {
     var sPageURL = window.location.search.substring(1);
     var sURLVariables = sPageURL.split('&');
-    for (var i = 0; i < sURLVariables.length; i++) 
+    for (var i = 0; i < sURLVariables.length; i++)
     {
         var sParameterName = sURLVariables[i].split('=');
-        if (sParameterName[0] == sParam) 
+        if (sParameterName[0] == sParam)
```

Comment:
Changes in this file are note necessary.

Reply:
there was no changes , could have been an auto formated that line..
however its ok.